assets/js/script.js

```diff
@@ -0,0 +1,386 @@
+// store all options due options being the same
+const options = ["Yes", "No"];
+
+//store answers in array
+const rightAnswer = ["Yes", "No"];
+
+// global declarations of questions
+const questions = [
+  {
+    questions: "Does HTML stand for HyperText Markup Language?",
+    options: ["Yes", "No"],
```

Comment:
You've declared options here but you've also put them at the top in a variable. Try and stick to one way of doing this.